### PR TITLE
[RI2] Minor updates to content

### PR DESCRIPTION
**Pull request by michaelspedersen**
* Updated references to Packet with Equinix Metal
* Added link to OVP 2.0 (lfnetworking wiki)
* Updated configuration references for Kuberef

### CODE REVIEW

File: doc/ref_impl/cntt-ri2/chapters/chapter04.md
Comment (tomkivlin) [inline doc/ref_impl/cntt-ri2/chapters/chapter04.md:48]: Q: is this removed because it's no longer required? i.e. I wouldn't need to modify that inventory.ini file any more?

**Review by tomkivlin — APPROVED** [review]

File: doc/ref_impl/cntt-ri2/chapters/chapter03.md
Comment (collivier) [inline doc/ref_impl/cntt-ri2/chapters/chapter03.md:19]: RI2 is verified by RC2 to onboard CNF.
RC2 shouldn't be developed on RI2 (it's the control loop

**Review by collivier — CHANGES_REQUESTED** [review]

**Comment by collivier** [comment]
It's quite unlinked to your update. I could open a bug if it helps.

**Comment by michaelspedersen** [comment]
> It's quite unlinked to your update. I could open a bug if it helps.
I think it would be better to do this in a separate PR, but I do agree that this needs to be updated

**Review by rihabbanday — APPROVED** [review]

File: doc/ref_impl/cntt-ri2/chapters/chapter03.md
Comment (tomkivlin) [reply doc/ref_impl/cntt-ri2/chapters/chapter03.md:19]: @collivier could you suggest some language that makes it clearer? (For me, the current wording just means we verify RI2 using RC2 as a means to validating that RC2 is delivering what is needed, not that RI2 is the development platform.)

Comment (michaelspedersen) [reply doc/ref_impl/cntt-ri2/chapters/chapter03.md:19]: @collivier @tomkivlin Should we create a ticket specifically for the above? This PR has been lingering for almost a month, so it would be good to get it merged

**Comment by tomkivlin** [comment]
@collivier could you remove the CR block on this please? Or just let me know if you're happy for me to merge and I'll force it through.

File: doc/ref_impl/cntt-ri2/chapters/chapter03.md
Comment (michaelspedersen) [reply doc/ref_impl/cntt-ri2/chapters/chapter03.md:19]: I've created #2125 to track the updated purpose related to RC2

**Comment by tomkivlin** [comment]
@collivier I'm going to force merge this tomorrow, unless you shout and tell me the changes you asked for haven't been addressed?

**Comment by michaelspedersen** [comment]
> 
> 
> @collivier I'm going to force merge this tomorrow, unless you shout and tell me the changes you asked for haven't been addressed?

Not sure it is addressed yet, but tracked through #2125. Not updating it in this PR is intentional

**Comment by tomkivlin** [comment]
@michaelspedersen could you resolve the conflicts please and then I'll merge.

**Comment by michaelspedersen** [comment]
> @michaelspedersen could you resolve the conflicts please and then I'll merge.

Done

**Pull request merged**
